internal/pkg/cli/svc_logs.go

```diff
@@ -351,7 +351,7 @@ func buildSvcLogsCmd() *cobra.Command {
 	cmd.Flags().IntVar(&vars.limit, limitFlag, 0, limitFlagDescription)
 	cmd.Flags().StringSliceVar(&vars.taskIDs, tasksFlag, nil, tasksLogsFlagDescription)
 	cmd.Flags().StringVar(&vars.logGroup, logGroupFlag, "", logGroupFlagDescription)
-	cmd.Flags().BoolVarP(&vars.previous, previousFlag, previousFlagShort, false, previousFlagDescription)
-	cmd.Flags().StringVar(&vars.containerName, containerLogFlag, "", containerLogFlagDescription)
+	// cmd.Flags().BoolVarP(&vars.previous, previousFlag, previousFlagShort, false, previousFlagDescription)
```

Comment:
We can probably use this feature https://pkg.go.dev/github.com/spf13/pflag#readme-hidden-flags to hide some flags!

Reply:
i considered that - i'm good either way. hidden flags they can still use the feature if wanted, this way they can't :) do you have a preference?

Reply:
Sounds like @paragbhingre is going to update the feature further, so maybe continue with the revert? Parag, can you confirm?

Reply:
an alternative would be putting everything behind a "featureFlag"
```
type svcLogsOpts struct {
   featureFlag bool
}

func (o *svcLogsOpts) Execute() error {
  if o.previous && o.featureFlag {}
  var container string
  if o.featureFlag {
    container = o.containerName
  }
  err := o.logsSvc.WriteLogEvents(logging.WriteLogEventsOpts{
		Follow:        o.follow,
		Limit:         limit,
		EndTime:       o.endTime,
		StartTime:     o.startTime,
		TaskIDs:       o.taskIDs,
		OnEvents:      eventsWriter,
		ContainerName: containerName,
	})
}
``` 

but I'm happy with either way! Just putting this out there!

Reply:
i'm in favor of just keeping it simple...

Reply:
I think what @dannyrandall has done is fine. I can cut a separate PR to undo Danny's changes after patch release and add display `taskId` feature that we discussed in the meeting.